conf/messages.en

```diff
@@ -399,9 +399,9 @@ pdf.application.section.other.heading = Other information about your item
 pdf.application.section.other.haveOtherInformation =Do you have supporting information?
 pdf.application.section.other.supportingInformation = Supporting information
 pdf.application.footer.whatHappensNext.heading = What happens next
-pdf.application.footer.whatHappensNext.html = <p>HMRC will issue your Binding Tariff Information ruling by email within 30 days unless you:</p><ul><li>are sending samples</li><li>are asked for more information to support your application</li></ul><p>You can find out the status of your application by signing into your BTI account. Go to GOV.UK and search for Binding Tariff Information (Notice 600).</p>
+pdf.application.footer.whatHappensNext.html = <span id="pdf.application.footer.whatHappensNext"><p>HMRC will issue your Binding Tariff Information ruling by email within 30 days unless you:</p><ul><li>are sending samples</li><li>are asked for more information to support your application</li></ul><p>You can find out the status of your application by signing into your BTI account. Go to GOV.UK and search for Binding Tariff Information (Notice 600).</p></span>
```

Comment:
Since when do we put html in message files?

Reply:
Been there for some time I think - was just updated for this ticket

Reply:
Longer answer is 2 months ago in this commit - https://github.com/hmrc/binding-tariff-trader-frontend/commit/2ca62615ebf54c9e8f1fa08a66351e4308b34de3

Reply:
Done